tree/tree/src/TTree.cxx

```diff
@@ -1209,6 +1209,18 @@ void TTree::AddClone(TTree* clone)
    }
 }
 
+void CheckIfReshuffled(TTree *t1, TTree *t2)
```

Comment:
We may want to now return a bool and have the caller *not* add the friend in that case.

Reply:
done